src/services/services.ts

```diff
@@ -9,7 +9,7 @@
 /// <reference path='formatting\smartIndenter.ts' />
 
 module ts {
-    export var servicesVersion = "0.4"
+    export var servicesVersion = "0.4.1"

```

Comment:
Bumping up the API version will require updating the check on the managed side as well